src/components/cavatica/CavaticaCopyButton.js

```diff
@@ -58,14 +60,14 @@ const showCopyModal = ({ effects, props }) => {
 };
 
 const CavaticaCopyButton = compose(injectState, withTheme)(
-  ({ state, theme, effects, disabled, ...props }) => {
+  ({ state, theme, effects, ...props }) => {
     const connected = state.integrationTokens[CAVATICA];
     const clickAction = connected ? showCopyModal : showConnectModal;
     return (
-      <CavaticaButton disabled={disabled} onClick={() => clickAction({ effects, props })}>
+      <CavaticaButton disabled={props.disabled} onClick={() => clickAction({ effects, props })}>
```

Comment:
Just curious, any reason to pull `disabled` out of the destructuring and looking it up on the `props` object?